newsfragments/30083.significant.rst

```diff
@@ -0,0 +1,8 @@
+Timezone-traversing non-fixed cron timetables properly accounts for timezone delta.
+
+Given a cron expression which results in an `interval` based schedule instead of a `fixed` schedule such as `0 9-16 * * *`, the Monday after a timezone change (e.g. Daylight Savings Time) the first job would be one hour offset.
+This was due to intentional code which accounted for timezone deltas, introduced after the switch to timetables (in `airflow>=2`).
+
+As an example, if operating in the `America/New_York` timezone when entering daylight savings time, the above schedule would result in a `10am` execution on Monday morning, instead of the previous `9am`.
+
+This functionality is now deprecated and reverted, so timezone-aware timetables will now run at the same time, e.g. a `9am` timetable will always run at `9am` local time.
```

Comment:
A deprecation would imply the functionality is still available but disfavoured. This is not the case here.